src/include/cell.i

```diff
@@ -58,7 +58,10 @@
  * Bit 3 marks an 8B packed, uint64_t value following the cell description byte.
  * (A run-length counter or a record number for variable-length column store.)
  *
- * Bit 4 is unused.
+ * Bit 4 marks a value with durable timestamps. For a start, stop value pair,
+ * it's the same as WT_TS_NONE and WT_TS_MAX. For an oldest_start, newest_start,
+ * newest_stop address triplet, it's the same as WT_TS_NONE, WT_TS_NONE and
+ * WT_TS_MAX.
```

Comment:
Did you consider flipping this, so bit 4 being on means that timestamps are present?  I don't feel strongly, that just seems to better match the existing format without timestamps.

Reply:
> Did you consider flipping this, so bit 4 being on means that timestamps are present? I don't feel strongly, that just seems to better match the existing format without timestamps.

I did not think of that (I added timestamps before thinking about durable timestamps being special). That approach makes things better because "short" values use 6 bits of the cell for the data length, which means the bit 0x08 isn't available to flag a special timestamp case. The way I did it, we can use short values if there are timestamps included, but we can't use short values if the timestamps are globally durable because we'll need bit 0x08 to flag the missing timestamps. In your approach, we get to use short values if there aren't timestamps included, and that should be the default case over time, that is, I expect more values without timestamps than with timestamps.

tl;dr
I didn't think of it, but I really like it.

Reply:
This change worked out really nicely, it's much better this way. Pushed in 3c3b841.